JS/app.js

```diff
@@ -35,89 +55,125 @@ Store.prototype.randomCookieValue = function () {
-  // attach content
-  
   tStoreNameEl.textContent = this.getStore();
-  
   tRowEl.appendChild(tStoreNameEl);
-  
+  // for generates cookie amounts over all hours
   for(var i = 0; i < hours.length; i++) {
-    //  create  elements
-    // function set to a variable 
-    var localRandomCookieValue= this.randomCookieValue();
-    allStores[i] =(allStores[i]+localRandomCookieValue);
-    //removed this from create element
     var tDataEl = document.createElement('td');
-    // attach content
-
-    tDataEl.textContent = localRandomCookieValue;
-    
-    // 3. append to the DOM
-    // parentElement.appendChild(childElement);
+    tDataEl.textContent = this.soldCookiesHour[i];
     tRowEl.appendChild(tDataEl);
-    // console.log('rando cookie hours: ', localRandomCookieValue);
-    // console.log( 'totalForAllHours ' , this.totalForAllHours);
-    this.totalForAllHours = (this.totalForAllHours + localRandomCookieValue);
   }
-  console.log('allstores at i: ', allStores[i]);
-  //Total cookie    
-  // 
+  // finishes with last column which is sum of hours
   var tTotalEl=document.createElement('td');
   tTotalEl.textContent = this.totalForAllHours;
   tRowEl.appendChild(tTotalEl);
-  //  
   tableEl.appendChild(tRowEl);
+};
+//All stores call
+var renderAllCookieRow = function() {
+  for(var i; i<5; i++) {
+    allStores[i].renderCookieRow();
+  }
+};
 
-  tRowEl = document.createElement('tr');
-  //sets blank cell
-  var tHoursRow = document.createElement('td');
-  tHoursRow.textContent = '';
-  tRowEl.appendChild(tHoursRow);
-  // fills in hours
-  for(i=0; i<hours.length; i++){
-    tHoursRow = document.createElement('td');
-    tHoursRow.textContent = allStores[i] ;
-    tRowEl.appendChild(tHoursRow);
+var renderCookieFooter= function() {
+  // creates first block in table(Hard coded)
+  tRowEl_2 = document.createElement('tr');
+  var tStoreNameEl = document.createElement('tfoot');
+  tStoreNameEl.textContent = "total for hour";
+  tRowEl.appendChild(tStoreNameEl);
+  // for generates cookie amounts over all hours
+  for(var i = 0; i < hours.length; i++) {
+    var tDataEl = document.createElement('tfoot');
+    tDataEl.textContent = totalArrPerHour [i];
+    tRowEl.appendChild(tDataEl_2);
   }
-  tHoursRow = document.createElement('td');
-  tHoursRow.textContent = allStores;
-  tRowEl.appendChild(tHoursRow);
-  //attach to table
+  // finishes with last column which is sum of hours
+  var tTotalEl=document.createElement('td');
+  tTotalEl.textContent = this.totalForAllHours;
+  tRowEl.appendChild(tTotalEl);
   tableEl.appendChild(tRowEl);
+  
+Store.addNewStore= function(event) {
+//prevent refreshing of page
+  event.preventDefault();
+  var newLocation = event.target.storeLocation.value;
+  var newMinSales = event.target.minimumSales.value;
+  var newMaxSales = event.target.maximumSales.value;
+  var newAvgSales = event.target.avgSales.value;
+
+  new Store(newLocation, newMinSales, newMaxSales, newAvgSales);
```

Comment:
I would try a console.log of the whole line 140 to see if this new object is being instantiated the way you intend.